### ISSUE TITLE
got an error  when run chatrtx windows 10

### ISSUE THREAD

**Issue by thalapandi** [NVIDIA/trt-llm-rag-windows#98]
Environment path found: C:\Users\jayme\AppData\Local\NVIDIA\ChatWithRTX\env_nvd_rag
App running with config
{
“models”: {
“supported”: [
{
“name”: “Mistral 7B int4”,
“installed”: true,
“metadata”: {
“model_path”: “model\mistral\mistral7b_int4_engine”,
“engine”: “llama_float16_tp1_rank0.engine”,
“tokenizer_path”: “model\mistral\mistral7b_hf”,
“max_new_tokens”: 1024,
“max_input_token”: 7168,
“temperature”: 0.1
}
},
{
“name”: “Llama 2 13B int4”,
“installed”: true,
“metadata”: {
“model_path”: “model\llama\llama13_int4_engine”,
“engine”: “llama_float16_tp1_rank0.engine”,
“tokenizer_path”: “model\llama\llama13_hf”,
“max_new_tokens”: 1024,
“max_input_token”: 3900,
“temperature”: 0.1
}
}
],
“selected”: “Mistral 7B int4”
},
“sample_questions”: [
{
“query”: “How does NVIDIA ACE generate emotional responses?”
},
{
“query”: “What is Portal prelude RTX?”
},
{
“query”: “What is important about Half Life 2 RTX?”
},
{
“query”: “When is the launch date for Ratchet & Clank: Rift Apart on PC?”
}
],
“dataset”: {
“sources”: [
“directory”,
“youtube”,
“nodataset”
],
“selected”: “directory”,
“path”: “dataset”,
“isRelative”: true
},
“strings”: {
“directory”: “Folder Path”,
“youtube”: “YouTube URL”,
“nodataset”: “AI model default”
}
}
Traceback (most recent call last):
File “C:\Users\jayme\AppData\Local\NVIDIA\ChatWithRTX\env_nvd_rag\lib\site-packages\langchain\embeddings\huggingface.py”, line 58, in init
import sentence_transformers
ModuleNotFoundError: No module named ‘sentence_transformers’

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
File “C:\Users\jayme\AppData\Local\NVIDIA\ChatWithRTX\RAG\trt-llm-rag-windows-main\app.py”, line 114, in
embed_model = HuggingFaceEmbeddings(model_name=embedded_model)
File “C:\Users\jayme\AppData\Local\NVIDIA\ChatWithRTX\env_nvd_rag\lib\site-packages\langchain\embeddings\huggingface.py”, line 61, in init
raise ImportError(
ImportError: Could not import sentence_transformers python package. Please install it with pip install sentence-transformers.
Press any key to continue . . .

when run chatrtx on windows 10 got this error sow how to resolve of an issue

System-specification:
Platform-windows 10
nvidia driver version - 560
cuda version - 12.5

**Comment by DrJaymz** [comment]
Yes, it no longer works as supplied.  On installation it installs some packages which are incompatible and trying to resolve them would appear to not be possible.

**Comment by DrJaymz** [comment]
> Yes, it no longer works as supplied. On installation it installs some packages which are incompatible and trying to resolve them would appear to not be possible.

I spent about 2 hours on this.  I haven't yet come across a package manager that actually doesn't get into circular dependency hell.  So the problem is that when you install it installs packages of later versions that it was originally built with, those then complain that others are not compatible, so you might think you just update those, but you can't because the later ones remove things it relied on.  If you just keep trying to satistfy the dependencies it still won't run because it then needs something that is depricated.  So you cannot resolve it with dependencies unless you have encyclopedic knowledge of everything it uses.  Since you cannot know this, it cannot be resolved.

The installer package should have had fixed requirements.txt since it didn't its now not possible to determine short of trying to work out the packages that were available on the date it used to work.

**Comment by akaanupkumar** [comment]
We have released new 0.5 version. We have not observed such issue in 0.5 version. Please verify the prerequisites and give that a try.
